packages/foam-vscode/static/graphs/default/graph.js

```diff
@@ -160,10 +167,10 @@ function initDataviz(channel) {
     .d3Force('x', d3.forceX())
     .d3Force('y', d3.forceY())
     .d3Force('collide', d3.forceCollide(graph.nodeRelSize()))
-    .linkWidth(0.2)
+    .linkWidth(() => model.style.lineWidth || styleFallback.lineWidth)
     .linkDirectionalParticles(1)
-    .linkDirectionalParticleWidth(link =>
-      getLinkState(link, model) === 'highlighted' ? 1 : 0
+    .linkDirectionalParticleWidth(
+      () => model.style.particleWidth || styleFallback.particleWidth
```

Comment:
does this mean that particles will be always visible? (as opposed to only for selected nodes)

Reply:
Good catch - that was embarassing that I didn't notice this when testing, thanks! Fixed shortly.